src/app/devices/sol/sol.component.spec.ts

```diff
@@ -36,7 +33,7 @@ describe('SolComponent', () => {
         provide: ActivatedRoute,
         useValue:
           { params: of({ id: 'guid' }) }
-      }, { provide: AuthService, useValue: authService }],
+      }, { provide: AuthService, useValue: { getLoggedUserToken: () => '' } }],
```

Comment:
actually, the way this was done before was better, if you wanna change it back, and then update KVM test as well, that would be helpful

Reply:
done. updated kvm test as well in kvm-integration branch